ocs_ci/deployment/tnf.py

```
@@ -0,0 +1,543 @@
+                logger.info("Step 2: Waiting for SSH access...")
+                hypervisor.wait_for_ssh_ready()
+
+                logger.info("Step 3: Configuring host as KVM hypervisor...")
+                hypervisor.configure_host()
+
+                logger.info("Step 4: Cloning dev-scripts...")
+                hypervisor.clone_dev_scripts()
+
+                logger.info("Step 5: Generating dev-scripts config...")
+                pull_secret_path = os.path.join(constants.DATA_DIR, "pull-secret")
+                with open(pull_secret_path, "r") as f:
```

Ruff (0.16.1)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)